superset-frontend/src/reduxUtils.ts

```diff
@@ -52,17 +67,22 @@ export function alterInArr(state, arrKey, obj, alterations, idKey = 'id') {
   return { ...state, [arrKey]: newArr };
 }
 
-export function removeFromArr(state, arrKey, obj, idKey = 'id') {
-  const newArr = [];
-  state[arrKey].forEach(arrItem => {
+export function removeFromArr(
+  state: Record<string, any>,
+  arrKey: string,
+  obj: Record<string, any>,
+  idKey = 'id',
```

Comment:
type `idKey`?

Reply:
i think type is auto inferred, but i can add a string type here

Reply:
actually, because the type is auto inferred from the default value, the linter auto removes it. so this should stay as is

Reply:
good to know this.